pkg/vsphere/extraconfig/keys.go

```diff
@@ -27,30 +27,73 @@ import (
 )
 
 const (
-	// DefaultTagName value
+	// GuestInfoPrefix is dictated by vSphere
+	GuestInfoPrefix = "guestinfo."
```

Comment:
Unrelated to this change, but it seems like this prefix is hard-coded elsewhere:

https://github.com/vmware/vic/blob/1387ab15f05282c466982415716e63a6f3c1591e/pkg/vsphere/extraconfig/secret.go#L30

https://github.com/vmware/vic/blob/1387ab15f05282c466982415716e63a6f3c1591e/lib/portlayer/exec/base.go#L147-L151

https://github.com/vmware/vic/blob/1387ab15f05282c466982415716e63a6f3c1591e/lib/migration/manager/manager.go#L35-L36

Should those be updated to refer to this prefix?

Reply:
The manager should be updated with a test that asserts that the key returned by `extraconfig.CalculateKey` matches the embedded string so we're alerted if we ever try to change the version location.

For the secrets, yes.

For `base.go` - this should be using `CalculateKey`